.tx/config

```diff
@@ -1,7 +1,7 @@
 [main]
 host = https://www.transifex.com
 
-[dash.qt-translation-018x]
+[dash.qt-translation-020x]
```

Comment:
18360: nit: actually, in our case it should really say
```suggestion
[dash.dash_ents]
```
if understand the purpose of this value correctly
https://app.transifex.com/dash/dash/dash_ents/

Reply:
I created a new PR, will rebase/update that when this one is merged.